src/librustdoc/html/markdown.rs

```diff
@@ -462,6 +467,11 @@ impl LangString {
         let mut seen_rust_tags = false;
         let mut seen_other_tags = false;
         let mut data = LangString::all_false();
+        let allow_compile_fail = if let UnstableFeatures::Cheat = get_unstable_features_setting() {
```

Comment:
@brson: This seems to not work. For what I understood, if it returned `UnstableFeatures::Cheat`, it means that unstable features were allowed. Did I understood wrong?